feature/security/gnsi/certz/server_certificate_rotation/README.md

```diff
@@ -0,0 +1,97 @@
+# X.X: Server Certificate Rotation
+
+## Summary
+
+Certificates on network devices (servers) must be rotated over time for various
+operational reasons. The ability to perform a rotation is a key component of
+safe operation practices.
+
+## Baseline Setup
+
+### Input Args
+
+   * the set of certificate testdata generated with the mk_cas.sh script
+   in featureprofiles/feature/certificate-authorities/
+
+### DIT Service Setup
+
+Configure the DUT to enable the following services (that are using gRPC) are up
+and required user mTLS for authentication:
+
+   * gNMI
+   * gNOI
+   * gNSI
+   * gRIBI
+
+Be prepared to load the relevant trust_bundle.pem file for each test Certificate
+Authority(CA) under test on the DUT. Each CA has an RSA and ECDSA form, both
+must be tested.
+
+## Tests
+
+### Certz-3.0
```

Comment:
Cert-3.x, x = [1,n]

Reply:
oh! haha, start at 1, do not start at zero.

Why no zero indexed? :)